### ISSUE TITLE
Propose new logo

### ISSUE THREAD

**Issue by anaufalm** [tengusw/share_to_clipboard#19]
Hello, I want to contribute to this awesome project, and I have a new logo design for you, if you allowed I will send to you :)

**Comment by tasn** [comment]
Sorry for the delay in response!
I'd love to have a look at the new logo. Maybe attach it to this ticket?

Thanks!

**Comment by naufaal** [comment]
Hi @tasn , this is me @anaufalm, I just want to inform you that I lost my account few weeks ago, so sad and I use this account for now, thank you. And sorry just responding to your message now, I was very busy last month with my studies. And now I can focus on contributing.

About your logo, I will make it asap, please wait. :)

**Comment by tasn** [comment]
No worries. I know how life gets. :)
Looking forward...

**Comment by tasn** [comment]
Thanks a lot! I love the concept of the second one, I think that one is the winner!

How does it look if you mirror the share part (so it's the same direction as the original share icon)? Just a thought, no idea if it's any good.

Additionally, I'm not a huge fan of gradients. Could you maybe do it in two colours? E.g. the pad in brown and the share button in black? or better yet, could you maybe put it in a circle/square like in Material design, have the square/circle brown/organge kind of (similar to existing colour) and the shape white (with a shadow?).
Thinking something like this icon with your shape https://play.google.com/store/apps/details?id=com.tengu.musicstudio&hl=en

Just a few random thoughts. Design is a very iterative process, at least for me, so no idea if they are any good, but let's try? :)

Thanks again!

**Comment by naufaldsg** [comment]
1. I have made a mirror version of the previous logo, and it turns out you are right they look more real.
2. I slightly increase the size of the object from before so that now they look stronger.
3. And in the first picture I made two versions, namely logo A with sharp angles, and logo B with corners that were blunt (round). What do you think?
4. Add version logo icon according to material design.
5. I am not sure of the colors you give (brown and black), but I suggest other colors that you might choose.

![share-02](https://user-images.githubusercontent.com/44208945/48059464-da402080-e1eb-11e8-8fce-235fdf78687b.png)

**Comment by tasn** [comment]
Thanks!

I think I like the middle one with the blue background on the lower row best. What do you think?

Also, look at the rounded corners, they look much more round than the material design guidelines see https://material.io/design/iconography/product-icons.html#grid-keyline-shapes

I'd say:
1. Adjust the corners to be like the guidelines.
2. If at all possible, make the inside of the square (the icon itself) slightly bigger, to cover more of the square (I know it's not strictly material design compatible, but I always found that part of their guidelines a bit small, and no one follows this one).
3. Send me an SVG if at all possible, if not, a very high rest png is fine.

After that, I think we are done. :)

Thanks again!

**Comment by naufaldsg** [comment]
Yaa I see, thanks for correction. It just an sample icon and I dont create based on material. 😂
All your assets file will be in the folder, wait for it ;)

A, B, C, D? wich one do you prefer? or combination between one of them?

![share-03](https://user-images.githubusercontent.com/44208945/48072194-872b9500-e20e-11e8-8057-525946d7d968.png)

**Comment by tasn** [comment]
Out of these, I prefer A, but I didn't mean these ones.

There is a triplet just above them in your previous comment, the one there in the middle. :)

**Comment by naufaldsg** [comment]
Sorry for delay response, this is the result:

![share-04](https://user-images.githubusercontent.com/44208945/48143751-0f2e9f00-e2e2-11e8-8f9f-e928ec558d7b.png)

This is icon that I've made bigger than material,
![share-05](https://user-images.githubusercontent.com/44208945/48143953-7f3d2500-e2e2-11e8-8d1e-24d7547e7160.png)

**Comment by tasn** [comment]
Good job! :tada: :tada: 

I think the middle one of the "bigger than material" icons is the best for the app icon.
I like the logo and the heading one with the text (forgot how it's called, grr). I'd make just one comment on that one though. Can you justify the text so "SHARE TO" is the same width as "CLIPBOARD"?

Thanks again, looking good!

**Comment by naufaldsg** [comment]
Sorry, I'll explain a little about this logotype. We can't use Option C because it's a big mistake. But, if you use option A or option B logotype it will look funny because it loses balance. I can't justify the text because it can damage text construction, so I can only suggest using the first logotype.
![share-06](https://user-images.githubusercontent.com/44208945/48148474-676a9e80-e2ec-11e8-932d-f67246164371.png)

**Comment by naufaldsg** [comment]
![share-07](https://user-images.githubusercontent.com/44208945/48148899-7140d180-e2ed-11e8-86bb-c5287183dc6a.png)

**Comment by tasn** [comment]
Yeah, you are right, it looks horrible. OK, this is the winner. :)

Thanks again, and good job! Please send me the SVGs and I'll add them (+ give you credit everywhere).

**Comment by naufaldsg** [comment]
I have uploaded it and make a pr, would you check it? If there is anything need improvement, feel free to contact me. :)

**Comment by tasn** [comment]
Thanks. Will take a look soon. :)
